core/templates/pages/blog-admin-page/blog-admin-page.module.ts

```diff
@@ -79,6 +81,11 @@ declare var angular: ng.IAngularStatic;
       provide: HAMMER_GESTURE_CONFIG,
       useClass: MyHammerConfig
     },
+    {
+      provide: ErrorHandler,
+      useClass: AppErrorHandlerWithFirebaseErrorFilter,
+      deps: [HttpClient, LoggerService]
+    },
```

Comment:
This seems to common across all the modules. How about you move this object to the [app-error-handler.ts](https://github.com/oppia/oppia/pull/15783/files#diff-c60aced9c5cd5fbb6e25b535a867307027290b24b90a9a04a2439bbdd75bcc8a) file and import it here.

Reply:
done